docs/howtos/breaking-changes.md

```diff
@@ -0,0 +1,22 @@
+# Breaking changes in application-services code
+
+Application-services components are consumed by multiple consumers including Firefox Android,
+Firefox iOS, Focus Android, and Focus iOS.  To minimize the disruption to those projects when making
+breaking API changes, we follow a simple rule: **Fix the breakage in the other repos before merging
```

Comment:
As a trivial nit, I'd suggest replacing "Fix" in this line with something a little more explicit - eg, "Have approved PRs ready to land which fix ..."